lib/rucio/core/permission/atlas.py

```diff
@@ -121,6 +122,7 @@ def has_permission(issuer, action, kwargs):
             'add_attribute': perm_add_account_attribute,
             'del_attribute': perm_del_account_attribute,
             'list_heartbeats': perm_list_heartbeats,
+            'send_heartbeats': perm_send_heartbeats,
```

Comment:
isn't "create_heartbeats" better suited? I'm not sure, just asking.

Reply:
heartbeats are created in services external to rucio. They are just sent to Rucio...